src/style.css

```diff
@@ -1 +1,82 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+/* :root {
+  --color-active: #00acdc;
+} */
+
+.header {
```

Comment:
Add `padding: 0 50px; `
It will give you  the correct inner space for your header and at the same time it will position your logo with nav
In this case we will clean up incorrect margins at logo and nav

Reply:
If I do this the test fails